invenio/templates/application.yaml

```diff
@@ -112,22 +101,83 @@ spec:
         volumeMounts:
           - name: uwsgi-config
             mountPath: '/opt/invenio/src/uwsgi'
-          - name: "invenio-assets"
-            mountPath: "{{ .Values.web.assets.location }}"
+          - name: nginx-invenio-assets
+            mountPath: /opt/nginx-invenio-assets
         resources:
           requests:
             memory: 200Mi
           limits:
             memory: 1Gi
+      - name: nginx
```

Comment:
Moderate: It has not resource limiting (see below line 145 for example on filebeat). Do you think it is needed? If so, do we know sane defaults? Otherwise, I agree on leaving as is and set further on the road if needed.